lumberjack.go

```diff
@@ -379,8 +383,9 @@ func (l *Logger) millRunOnce() error {
 	}
 
 	if l.Compress {
-		for _, f := range files {
-			if !strings.HasSuffix(f.Name(), compressSuffix) {
+		for i, f := range files {
+			fileIndex := i
```

Comment:
No need for the additional variable.
```suggestion
		for fileIndex, f := range files {
```

Reply:
fixed